### DISCUSSION TITLE
How can I see the list of events?

### DISCUSSION THREAD

**Discussion by achikin** [Day8/re-frame-10x#292] [Q&A] [answered]
How can I see the list of events as I did in re-frisk?

**Comment by danielcompton** [comment, marked as answer]
To do this:

1. Go to the Trace tab
2. Select to show only events
3. Disable the "Only show traces for this epoch?" checkbox.

<img width="536" alt="screenshot of google chrome 4-02-19 11-11-45 am" src="https://user-images.githubusercontent.com/811954/52183551-bbded500-286d-11e9-8649-be7b85e08b97.png">

I think that's what you're looking for? Let me know if I've got it wrong and you're thinking of something else.
